firefox-ios/Storage/Rust/RustFirefoxSuggest.swift

```diff
@@ -25,8 +25,12 @@ public protocol RustFirefoxSuggestActor: Actor {
 public actor RustFirefoxSuggest: RustFirefoxSuggestActor {
     private let store: SuggestStore
 
-    public init(databasePath: String, remoteSettingsConfig: RemoteSettingsConfig? = nil) throws {
-        store = try SuggestStore(path: databasePath, settingsConfig: remoteSettingsConfig)
+    public init(dataPath: String, cachePath, remoteSettingsConfig: RemoteSettingsConfig? = nil) throws {
```

Comment:
You'll need to specifiy something here for the cache path. It can't just be an empty parameter.